### PR TITLE
[bitnami/prestashop] Follow non-root approach

### PR DESCRIPTION
**Pull request by marcosbc**
**Description of the change**

We are working on releasing the PrestaShop Docker image with a non-root approach. As part of that, we need to update the chart to use the new port numbers used by the container image and related settings.

We're also taking advantage of this PR being a major change by adding all standarizations (i.e. customizable command, extra volumes, extra envs, K8s labels standarization, etc.).

**Benefits**

Support for environments such as OpenShift, where a non-root UID is required to run a container.

**Possible drawbacks**

Breaking change with previous chart versions, as the default HTTP/HTTPS ports are changed to 8080/8443 (previously 80/443). The labels may also cause this to be a breaking change.

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/prestashop/README.md [hunk 1]
```diff
@@ -48,96 +48,142 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `tolerations`                        | Tolerations for pod assignment                                                                                        | `[]` (The value is evaluated as a template)    |
+| `updateStrategy`                     | Deployment update strategy                                                                                            | `nil`                                          |
+
+### Traffic Exposure Parameters
+
+| Parameter                        | Description                           | Default            |
+|----------------------------------|---------------------------------------|--------------------|
+| `service.type`                   | Kubernetes Service type               | `LoadBalancer`     |
+| `service.port`                   | Service HTTP port                     | `80`               |
+| `service.httpsPort`              | Service HTTPS port                    | `443`              |
```
Comment (carrodher) [inline bitnami/prestashop/README.md:137]:
```suggestion
| `service.port`                   | Service HTTP port                     | `8080`             |
| `service.httpsPort`              | Service HTTPS port                    | `8443`             |
```

File: bitnami/prestashop/values.yaml [hunk 2]
```diff
@@ -288,15 +430,23 @@ readinessProbe:
   failureThreshold: 6
   successThreshold: 1
 
+customLivenessProbe: {}
```
Comment (carrodher) [inline bitnami/prestashop/values.yaml:433]:
```suggestion
## Custom Liveness probe
##
customLivenessProbe: {}
```

File: bitnami/prestashop/values.yaml [hunk 3]
```diff
@@ -288,15 +430,23 @@ readinessProbe:
...
+
+customReadinessProbe: {}
```
Comment (carrodher) [inline bitnami/prestashop/values.yaml:435]:
```suggestion
## Custom Readiness probe
##
customReadinessProbe: {}
```

**Review by carrodher — APPROVED** [review]
LGTM, just a couple of minor suggestions.

If everything is working in the release pipeline is fine to merge the PR **once the new image is updated in this PR**

File: bitnami/prestashop/README.md (hunk 1 above)
Comment (marcosbc) [reply bitnami/prestashop/README.md:137]: Good catch, fixed

File: bitnami/prestashop/values.yaml (hunk 2 above)
Comment (marcosbc) [reply bitnami/prestashop/values.yaml:433]: fixed

File: bitnami/prestashop/values.yaml (hunk 3 above)
Comment (marcosbc) [reply bitnami/prestashop/values.yaml:435]: fixed

**Pull request merged**